server/aws-lsp-codewhisperer/src/language-server/chat/chatTelemetryController.ts

```diff
@@ -0,0 +1,90 @@
+import { MetricEvent } from '@aws/language-server-runtimes/server-interface'
+
+import { ChatTelemetryEventMap, ChatTelemetryEventName } from '../telemetry/types'
+import { Features } from '../types'
+import { isObject } from '../utils'
+
+export const CONVERSATION_ID_METRIC_KEY = 'CWSPRChatConversationId'
+
+export enum ChatUIEventName {
```

Comment:
I can rename these to `EnterUIFocus` `ExitUIFocus` - should be more clear

Reply:
UI focus can be even more confusing to be honest because in total there are 3 different focus events and it is already confusing to me, I would rather not add to the confusion and follow the same naming so I would suggest we explicitly name these `enterFocusConversation` and `exitFocusConversation`

Reply:
I think these are the `enterFocusChat` and `exitFocusChat` events, would it be okay if I rename those as those?